### ISSUE TITLE
Problems with MySQL adapter in 3.5

### ISSUE THREAD

**Issue by beatmadsen** [phoenixframework/phoenix_ecto#105]
When I'm setting up a new phoenix application with a MySQL database, either by using the `--database mysql` cli arg or by updating the dependencies and configurations manually following the guide for Phoenix v.1.3.4 I run into the following compilation issue:
```
== Compilation error in file lib/flurmex/repo.ex ==
** (ArgumentError) adapter Ecto.Adapters.MySQL was not compiled, ensure it is correct and it is included as a project dependency
    lib/ecto/repo/supervisor.ex:71: Ecto.Repo.Supervisor.compile_config/2
    lib/flurmex/repo.ex:2: (module)
    (stdlib) erl_eval.erl:680: :erl_eval.do_apply/6
    (elixir) lib/kernel/parallel_compiler.ex:206: anonymous fn/4 in Kernel.ParallelCompiler.spawn_workers/6
```
I'm on the latest homebrew MariaDB (10.3.10).

The compilation problem goes away when I downgrade `phoenix_echo` to  `3.4.0`.

Here's the output of `mix.deps` for the broken setup:
```
* file_system 0.2.6 (Hex package) (mix)
  locked at 0.2.6 (file_system) fd4dc3af
  ok
* connection 1.0.4 (Hex package) (mix)
  locked at 1.0.4 (connection) a1cae722
  ok
* gettext 0.16.0 (Hex package) (mix)
  locked at 0.16.0 (gettext) 4a7e9040
  ok
* ranch 1.3.2 (Hex package) (rebar3)
  locked at 1.3.2 (ranch) e4965a14
  ok
* decimal 1.5.0 (Hex package) (mix)
  locked at 1.5.0 (decimal) b0433a36
  ok
* poison 3.1.0 (Hex package) (mix)
  locked at 3.1.0 (poison) d9eb6366
  ok
* db_connection 1.1.3 (Hex package) (mix)
  locked at 1.1.3 (db_connection) 89b30ca1
  ok
* ecto 3.0.0 (Hex package) (mix)
  locked at 3.0.0 (ecto) 059250d9
  ok
* phoenix_pubsub 1.1.1 (Hex package) (mix)
  locked at 1.1.1 (phoenix_pubsub) 6668d787
  ok
* cowlib 1.0.2 (Hex package) (rebar3)
  locked at 1.0.2 (cowlib) 9d769a1d
  ok
* cowboy 1.1.2 (Hex package) (rebar3)
  locked at 1.1.2 (cowboy) 61ac29ea
  ok
* mime 1.3.0 (Hex package) (mix)
  locked at 1.3.0 (mime) 5e8d45a3
  ok
* mariaex 0.8.4 (Hex package) (mix)
  locked at 0.8.4 (mariaex) 5dd42a60
  ok
* plug_crypto 1.0.0 (Hex package) (mix)
  locked at 1.0.0 (plug_crypto) 18e49317
  ok
* plug 1.7.1 (Hex package) (mix)
  locked at 1.7.1 (plug) 8516d565
  ok
* phoenix 1.3.4 (Hex package) (mix)
  locked at 1.3.4 (phoenix) aaa1b55e
  ok
* phoenix_live_reload 1.1.7 (Hex package) (mix)
  locked at 1.1.7 (phoenix_live_reload) 425fff57
  ok
* phoenix_html 2.12.0 (Hex package) (mix)
  locked at 2.12.0 (phoenix_html) 1fb3c2e4
  ok
* phoenix_ecto 3.5.0 (Hex package) (mix)
  locked at 3.5.0 (phoenix_ecto) f72ec302
  ok
```

**Comment by josevalim** [comment]
Good catch. It should be fixed now as we retired phoenix_ecto 3.5.0 and released 4.0.0 instead.

**Comment by josevalim** [comment]
Thanks!
